book-2-mvc/chapters/ASPNET_INTRO.md

```diff
@@ -1,69 +1,81 @@
 
 ## Configuration
 
-Open your `appsettings.json` file and add your connection string. If you copy the text below, make sure you change it to contain your server name.
+Open the `appsettings.json` file and add your connection string. The file should look like this
 
 ```json
-"ConnectionStrings": {
-    "DefaultConnection": "Server=localhost\\SQLEXPRESS;Database=StudentExercises;Trusted_Connection=True;"
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=localhost\\SQLEXPRESS;Database=DogWalkerMVC;Trusted_Connection=True;"
+  }
 }
 ```
 
 ## Models
 
-You are going to be using the same models that you used in your API project. You can copy pasta those files into your new project.
+Create a `Neighborhood.cs` and `Walker.cs` file in the Models folder and add the following code
 
-![copying models from API project to MVC project](./images/DqeK4qXjRr.gif)
+> Neighborhood.cs
+```csharp
+public class Neighborhood
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+}
+```
 
+> Walker.cs
+```csharp
+public class Walker
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+    public string ImageUrl { get; set; }
+    public int NeighborhoodId { get; set; }
+    public Neighborhood Neighborhood { get; set; }
+}
+```
 
 ## Controller
 
-Time to use Visual Studio scaffolding to create a base controller for our **`Student`** resource. Right click on the `Controllers` directory in Solution Explorer. Then choose `Add > New scaffolded item`.
-
-In the window that appears, choose _MVC Controller with read/write actions_. Then click the _Add_ button. You will then be asked to name your controller, so name it `StudentsController`.
-
- Your controller will be automatically created for you and `StudentsController.cs` will appear in your Solution Explorer.
-
-![scaffolding a new controller in Visual Studio](./images/lsq7qz3ZAE.gif)
+We can use Visual Studio to scaffold us the skeleton of a controller. Right click on the Controllers folder in Solution Explorer and click Add > Controller > MVC Controller with Read/Write actions. Give it the name `WalkersController`
 
-### Injecting Configuration for Connection String
+Visual Studio kindly just created a whole bunch of code for us.
 
-Next, you can copy the controller constructor and `Connection` property from a previous project and rename the constructor to `StudentsController`. You can also copy it from below and paste it into your controller.
+Add a private field for `IConfiguration _config`, a constructor, and a computed `SqlConnection` property to the `WalkersController`
 
-You'll have to use the helpful lightbulb several times to get all of the packages imported.
```

Comment:
I think we should replace this line with something about `using Microsoft.Data.SqlClient`